tests/test_taar_lite_guidguid.py

```diff
@@ -0,0 +1,135 @@
+        ),
+        (
+            Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-3', 'test-guid-4']),
+            [Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-2', 'test-guid-3', 'test-guid-4']),
+             Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1', 'test-guid-5', 'test-guid-6']),
+             Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1'])]
+        ),
+        (
+            Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1', 'test-guid-5', 'test-guid-6']),
+            [Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-2', 'test-guid-3', 'test-guid-4']),
+             Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-3', 'test-guid-4']),
+             Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1'])]
+        ),
+        (
+            Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1']),
+            [Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-2', 'test-guid-3', 'test-guid-4']),
+             Row(key_addon='test-guid-1', coinstalled_addons=['test-guid-3', 'test-guid-4']),
+             Row(key_addon='test-guid-2', coinstalled_addons=['test-guid-1', 'test-guid-5', 'test-guid-6'])]
+        )]
+
+MOCK_KEYED_ADDONS = [
+    Row(key_addon='test-guid-1',
+        coinstalled_addons=['test-guid-2', 'test-guid-3', 'test-guid-4']),
+    Row(key_addon='test-guid-1',
+        coinstalled_addons=['test-guid-3', 'test-guid-4']),
+    Row(key_addon="test-guid-2",
+        coinstalled_addons=['test-guid-1', 'test-guid-5', 'test-guid-6']),
+    Row(key_addon="test-guid-2",
+        coinstalled_addons=['test-guid-1'])
+    ]
+
+
+EXPECTED_GUID_GUID_DATA = [
+    Row(key_addon=u'test-guid-2',
+        coinstallation_counts=[Row(id=u'test-guid-6', n=1),
+                               Row(id=u'test-guid-5', n=1),
+                               Row(id=u'test-guid-3', n=1),
+                               Row(id=u'test-guid-1', n=1)]),
+    Row(key_addon=u'test-guid-4',
+        coinstallation_counts=[Row(id=u'test-guid-1', n=1)]),
+    Row(key_addon=u'test-guid-3',
+        coinstallation_counts=[Row(id=u'test-guid-2', n=1), Row(id=u'test-guid-1', n=2)]),
+    Row(key_addon=u'test-guid-5',
+        coinstallation_counts=[Row(id=u'test-guid-6', n=1), Row(id=u'test-guid-2', n=1)]),
+    Row(key_addon=u'test-guid-1',
+        coinstallation_counts=[Row(id=u'test-guid-2', n=1),
+                               Row(id=u'test-guid-1', n=2),
+                               Row(id=u'test-guid-3', n=2),
+                               Row(id=u'test-guid-4', n=1)]),
+    Row(key_addon=u'test-guid-6',
+        coinstallation_counts=[Row(id=u'test-guid-2', n=1), Row(id=u'test-guid-5', n=1)])]
+
+
+@mock.patch('mozetl.taar.taar_lite_guidguid.extract_telemetry',
+            return_value=MOCK_TELEMETRY_SAMPLE)
+@mock_s3
+def test_extract_telemetry(spark):
```

Comment:
This test is safe to omit since it's not testing anything.

Reply:
Test dropped.